res/field/meson.build

```diff
@@ -13,3 +14,213 @@ tutorable_moves_h = custom_target('tutorable_moves.h',
         '--output-dir', meson.current_build_dir(),
     ]
 )
+
+zukan_enc_platinum_srcs = files(
```

Comment:
**suggestion**: If you get rid of all the files in `encounters_ex`, then move all of this build logic into the `encounters` folder.